app/views/courses/settings.html.erb

```diff
@@ -334,8 +334,8 @@
         </td>
       </tr>
       <tr>
-        <td class="form-label"><%= f.label :license, :en => "License" %></td>
-        <td colspan="3">
+        <td class="form-label"><label for="license"><%= before_label('license', %{License}) %></label></td>
+        <td colspan="3" id="license">
```

Comment:
Technically this change isn't needed because there's already an id on the select element (which is where the label for should point)